backend/inspirehep/pidstore/minters/doi.py

```diff
@@ -18,4 +18,4 @@ def create(self, pid_value):
 
     def get_pid_values(self):
         pid_values = {str(pid_value).lower() for pid_value in super().get_pid_values()}
-        return pid_values
+        return set(pid_values)
```

Comment:
isn't already a set?

Reply:
:+1:  Yes, you are right. Here it's already set.